src/BizHawk.Emulation.Cores/Consoles/Atari/Stella/LibStella.cs

```diff
@@ -28,7 +28,7 @@ public abstract bool stella_init(
 			[In] InitSettings settings);
 
 		[BizImport(CallingConvention.Cdecl)]
-		public abstract void stella_frame_advance(int port1, int port2, bool reset, bool power, bool leftDiffToggled, bool rightDiffToggled);
+		public abstract void stella_frame_advance(int port1, int port2, int switchPort, bool power);

```

Comment:
This does not match the unmanaged signature and the original one doesn't match either, how does this even work? Anyway, might be cleaner to make this a `[Flags]` enum considering the core-side is also manually parsing this instead of relying on actual input port data.

Reply:
Apparently it works because these arguments are just put in registers which are 8 bytes, so you could also declare every argument as `ulong`, but it doesn't mean you should. To be fair this was wrong before you came here but it still needs to be fixed: `uint8_t, uint8_t, uint8_t, bool` is `byte, byte, byte, bool`.

Reply:
Oh, apologies! Yeah I was just working off what was already there; didn't realize `byte` was the type that should've been used.